Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -235,8 +235,8 @@
         
         <dependency>
             <groupId>me.fhir</groupId>
-            <artifactId>fhir-0.81</artifactId>
-            <version>2382-SNAPSHOT</version>
+            <artifactId>fhir-0.12</artifactId>
```

Comment:
Not sure why this is showing up on this pr, considering this change is already in the code base.